pkg/dist/format-url.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Dist should only include the compiled index.js file. I think these belong in src?

Reply:
In order to use the package I have to set the compiler field composite to true which then automatically adds these to the dist.